contacts24/contacts.py

```diff
@@ -33,7 +34,7 @@ def parse_input(user_input: str) -> tuple[str, Optional[CommandArguments]]:
 
 def load_contacts_book() -> AddressBook:
```

Comment:
@krisdzhus please make the implementation the same as in `address_book.py` where we have filename as input

@IhorNi FYI